README.adoc

```diff
@@ -105,11 +103,6 @@ Enter to keep the current selection[+], or type selection number: 2
 - Maven >= 3.5 is required, download and extract if not installed using
 distribution package management, and add to PATH.
 
-Fedora 30
```

Comment:
I thought a bout leaving it as you say, without the mention to the OS. But then again, is just an install command, didn't add much value to the text above.

But I'm not against what you suggested, I could bring it back and remove the OS.

Reply:
I suggest that you remove the OS line, but leave the `maven` install command.
Then we can merge your changes and I will take it from there.

Reply:
I think it can be removed, the `maven` is already installed in the `Prerequisites` section above.
Although here it's installed as a `module`... @bennyz was there any reason for additional `maven` install as a module rather then a package?

Reply:
It is probably a leftover from when installing maven-3 was required, but it needs to be approved by infra

Reply:
@aesteve-rh So the removal is ok, also remove the comment on lines 105-107, Maven >= 3.5 is already required in `Prerequisites` section at the beginning.
And we'll let infra group to code review :)

Reply:
Done!